idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -22,55 +22,59 @@ export default `
       <div class="details-content">
         <div class="details-overview">
           <img src="{{thumbnail}}" width="260">
-          <section>
+          {{#model.measurements}}
```

Comment:
It seems weird to pass a model reference into the templates in this way.

The approach I had been using was to manually copy state from model into view, inside the view's `render` method, then pass the view itself into the `mustache.render(template, context)` call as the context object, for example, [here](https://github.com/mozilla/idea-town/blob/48cd247/idea_town/frontend/static-src/app/views/base-view.js#L11). This approach is verbose, but you always know that the `render` will explain where the template vars come from, as seen in [this view](https://github.com/mozilla/idea-town/blob/48cd247/idea_town/frontend/static-src/app/views/experiment-page.js#L21-L25), for example.

If you'd rather use the model directly as the data object for the view, maybe do that throughout, or at least throughout the experiment-page template. Another option might be to explicitly override the BaseView's `template` method, passing `this.model` into the `mustache.render` call. That would let you avoid writing `model.foo` everywhere for keys that are top-level in the model.

Ultimately, I think it's good for the views to be consistent, but I'll leave it to you to define how that should work :-)